frontend/src/modules/vehicles/VehicleProperties.vue

```diff
@@ -151,28 +128,29 @@ export default {
         .then((response) => {
           if (response.data.success) {
             this.vehicleProperties = response.data.payload;
-            this.splitVehicleProperties();
+            //this.splitVehicleProperties();
           }
         })
         .catch((err) => {
           console.log(err);
+          this.$router.replace("/pages/404");
         });
     },
-    splitVehicleProperties() {
-      let leftProperties = [];
-      let rightProperties = [];
-      this.vehicleProperties.forEach((property, index) => {
-        if (index % 2 == 0) {
-          // leftProperties
-          leftProperties.push(property);
-        } else {
-          // rightProperties
-          rightProperties.push(property);
-        }
-      });
-      this.leftProperties = leftProperties;
-      this.rightProperties = rightProperties;
-    },
+    // splitVehicleProperties() {
+    //   let leftProperties = [];
+    //   let rightProperties = [];
+    //   this.vehicleProperties.forEach((property, index) => {
+    //     if (index % 2 == 0) {
+    //       // leftProperties
+    //       leftProperties.push(property);
+    //     } else {
+    //       // rightProperties
+    //       rightProperties.push(property);
+    //     }
+    //   });
+    //   this.leftProperties = leftProperties;
+    //   this.rightProperties = rightProperties;
+    // },
```

Comment:
Remove this commented code

Reply:
fixed in the commit https://github.com/purplesmurf1998/Autotracks/pull/79/commits/848e7d87ac066714eebcfb9b63a370fc697ff283